Fix error for opencv 3.4.9+ over IplImage (#1949)

jeffeDurand opened the pull request.

davisking commented:
Huh, I've used OpenCV 4 for instance, and a few other versions newer than what this diff sets things to, and it always worked. Near as I could tell, only the very newest OpenCV revision on github needs this.  How did you get it to trip on older versions?

jeffeDurand commented:
Hi, opencv 3.4.9 is the newest version and introduced the same problem has
with 4.x. (issue 1949). I needed a fix I introduced in it on december and
was surprised by the dlib error in the opencv header.

I used the official revision macro (9+) to only allow newer version of the
3.4.x branch from now on.

I hope this satisfy your curiosity!

JF




Le mar. 7 janv. 2020 21 h 12, Davis E. King <notifications@github.com> a
écrit :

> Huh, I've used OpenCV 4 for instance, and a few other versions newer than
> what this diff sets things to, and it always worked. Near as I could tell,
> only the very newest OpenCV revision on github needs this. How did you get
> it to trip on older versions?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/davisking/dlib/pull/1963?email_source=notifications&email_token=AECEWRICPD3PGPIGZIOWLBTQ4UZAXA5CNFSM4KDKSXN2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOEIK6HUA#issuecomment-571859920>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AECEWRIGBSQER7WIBYMLBJDQ4UZAXANCNFSM4KDKSXNQ>
> .
>

davisking commented:
Thanks.  I didn't realize they were pushing the same change back into the 3.x versions.
